lib/tlscontext.c

```diff
@@ -697,6 +675,50 @@ tls_context_load_key_and_cert(TLSContext *self)
   return TLS_CONTEXT_OK;
 }
 
+static void _write_line_to_keylog_file_mutex(const char *file_path, const char *line)
```

Comment:
I think it's not necessary to state that this function locks the keylog_file mutex.
I'd convert this to a `TLSContext` method, and just simply lock `self->keylog_file_lock`.